tests/test_playbooks/scc_account.yml

```diff
@@ -0,0 +1,72 @@
+---
+- hosts: fixtures
+  gather_facts: false
+  vars_files:
+    - vars/server.yml
+  tasks:
+    - include: tasks/scc_account.yml
```

Comment:
Can you add setup and cleanup tasks for the 'Defaut Organisation' (and maybe rename it to 'Test Organization')?